### ISSUE TITLE
Redirects to do during PI13 Innovation 2

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#4321]
### Summary

**What we are after:**
_We are getting off of transition. Old files need to be redirected to their new homes before we can take 'em down._

### Related issues

- #4123 - Random press office documents (in this case the tax checkoff chart) - transition
- #4340 - Press statistic surveys - transition
- Accessibility redirects - see spreadsheet - www - these have been moved to https://github.com/fecgov/fec-proxy/pull/240

### Completion criteria

- [x] Redirects are merged into the release
- [x] 2021 spreadsheet is updated
- [x] Testing ticket is created to be done in the next sprint after deploy of releases. Also will need to check www redirect after release deploy.

### Future work
Testing after release is deployed

**Comment by dorothyyeager** [comment]
Transition redirects have been rolled into https://github.com/fecgov/fec-proxy/pull/239
